assets/css/list-widget.css

```
@@ -0,0 +1,201 @@
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.mform-list-widget.mform-list-widget-medialist.is-grid-view .mform-list-items li {
+    white-space: normal;
+    min-height: 48px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: .45rem;
+    word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)